scripts/x-charts.exports.json

```diff
@@ -64,6 +72,7 @@
   { "name": "getAreaElementUtilityClass", "kind": "Function" },
   { "name": "getAxisHighlightUtilityClass", "kind": "Function" },
   { "name": "getAxisUtilityClass", "kind": "Function" },
+  { "name": "getChartsLegendUtilityClass", "kind": "Function" },
```

Comment:
Nitpick: WDYT about removing `Charts` from this function to align with other instances of such functions? 🤔